components/textbox.tsx

```diff
@@ -2,29 +2,27 @@ import * as React from 'react';
 import styles from "../styles/Textbox.module.css";
 import Pointsbox from "./pointsbox";
 import {useState} from "react";
+import {AssessmentType} from "../types/Types";
+
+
 
 
-export interface Option {
-    value: number;
-    label: string;
-}
 interface textboxprop {
-    text: string |undefined;
-    maxPoints: number;
+    assessment: AssessmentType;
+    setAssessment: (assessment: AssessmentType, newScore: number) => void;
```

Comment:
now we finally know how to write this type! thanks peder, by any 😝

Reply:
Woop!!